lib/LogicalEquivalence/Circuit.cpp

```diff
@@ -120,6 +121,8 @@ void Solver::Circuit::performConcat(Value result, OperandRange operands) {
 
 void Solver::Circuit::performDivS(Value result, Value lhs, Value rhs) {
   LLVM_DEBUG(lec::dbgs() << name << " perform DivS\n");
+  allocateValue(lhs);
+  allocateValue(rhs);
   lec::Scope indent;
   LLVM_DEBUG(lec::dbgs() << "lhs:\n");
   z3::expr lhsExpr = fetchExpr(lhs);
```

Comment:
nit: Is it possible to perform allocation within `fetchExpr`? I kind feel it's a bit tedious to manually allocate values before uses.

Reply:
Ah, that would be much nicer! I was also thinking it seemed tedious but putting it in `fetchExpr` didn't occur to me, I'll get that updated. Thanks!